### PR TITLE
[mailbox/email] new designs + mobile view

### PR DESCRIPTION
**Pull request by isabellaenriquez**
Implemented new UI for Email and Mailbox, including: 
- mobile view

- new desktop view

- dropdown email tooltip for sender and receiver contacts in expanded email view

- date formatting

- new `click_action` for `Email` --> "mailbox" --> when selected, clicking condensed threads/messages open a separate view (rather than expand within the mailbox view; see 2nd and 4th screenshots in "After" column)

- updated Cypress test for `Email` to reflect new UI

# Before and After
| Before | After |
|---|---|
| Screen Shot 2021-07-29 at 3 23 18 PM | image |
| Screen Shot 2021-07-29 at 3 22 24 PM | image |
| Screen Shot 2021-07-29 at 3 25 11 PM | Screen Shot 2021-07-29 at 3 24 47 PM |
| Screen Shot 2021-07-29 at 3 25 32 PM | Screen Shot 2021-07-29 at 3 25 54 PM |
| image |  image |

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by wonka-lighthouse — COMMENTED** [review]
| Device | URL |
|--|--|
| mobile | http://localhost:8000/components/agenda/src/index.html |

     
| Device | URL |
|--|--|
| mobile | http://localhost:8000/components/composer/src/index.html |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by isabellaenriquez** [comment]
@srishaank I couldn't tag you as a reviewer so just going to tag you here!

**Review by philrenaud — CHANGES_REQUESTED** [review]
A few spacing revisions requested:

- feels weird to have everything centered while still collapsed here
CleanShot 2021-07-29 at 16 56 06

- "expanding your message" is present on un-opened messages when it oughtn't be:
CleanShot 2021-07-29 at 16 57 40

- whitespace between email items is taller than it should be:
CleanShot 2021-07-29 at 16 58 27

**Comment by isabellaenriquez** [comment]
@philrenaud thanks for pointing those out! for the first one, I attached a version where the lines go all the way as in the Figma to this message, but @srishaank what are you thoughts on the alignment?
Screen Shot 2021-07-29 at 5 35 28 PM

**Comment by srishaank** [comment]
Poked through some other products and I think this is our best bet. Maintain a 24pt margin once you get to wider desktop breakpoints.


Screenshot 2021-07-30 at 11 50 27 AM

**Review by philrenaud — APPROVED** [review]

File: components/mailbox/src/Mailbox.svelte [hunk 1]
```diff
@@ -140,11 +142,13 @@
   async function threadClicked(event: CustomEvent) {
     // console.debug('thread clicked from mailbox', event.detail);
     if (event.detail.thread?.expanded) {
+      openedEmailData = event.detail.thread;
       let message = await fetchIndividualMessage(
         event.detail.thread.messages[event.detail.thread.messages.length - 1],
       );
       threads = await MailboxStore.hydrateMessageInThread(message, query);
     }
+    console.log(openedEmailData);
```
Comment (pooja169usp) [inline components/mailbox/src/Mailbox.svelte:151]: remove log 🔪

**Review by pooja169usp — APPROVED** [review]

**Pull request merged**